src/services/offrampingFlow.ts

```diff
@@ -180,11 +187,13 @@ export async function constructInitialState({
     pendulumNode,
   );
 
-  const inputTokenDecimals = getInputTokenDetailsOrDefault(network, inputTokenType).decimals;
+  const { decimals: inputTokenDecimals, pendulumDecimals } = getInputTokenDetailsOrDefault(network, inputTokenType);
   const outputTokenDecimals = OUTPUT_TOKEN_CONFIG[outputTokenType].decimals;
 
   const inputAmountBig = Big(amountIn);
   const inputAmountRaw = multiplyByPowerOfTen(inputAmountBig, inputTokenDecimals || 0).toFixed();
+  const pendulumAmountRaw = multiplyByPowerOfTen(inputAmountBig, pendulumDecimals || 0).toFixed();
```

Comment:
We were previously using `inputAmountRaw` for all operations on Pendulum after writing the off-ramping state. Since the decimals on the starting chain and `axlUSDC` on Pendulum where the same (`6`), there was no issue. 

BSC uses 18 decimals so we need to account for this, or potentially for a different token combination.